docs/pages/guides/environment-variables.mdx

```diff
@@ -12,7 +12,9 @@ Environment variables are key-value pairs configured outside your source code th
 
 ## Using the `extra` field
 
-In the app config, there is an `extra` property. This property is available when you publish your project with EAS Update and when you build with EAS Build.
+In the app config, there is an `extra` property. If your variables are non-sensitive and can be checked into source code, you can just add them to the app.json `extra` property for local builds.
+
+For EAS builds, also add your environment variables to build.production.env in eas.json.
```

Comment:
```suggestion
```

I'd remove this for now. The page provides the link in the guide above on using env vars in the **eas.json** file. Repeating the info without the complete context (not everyone coming to his page will be familiar with EAS) might lead to confusion.